INSTALL.Unix.md

```diff
@@ -63,8 +63,8 @@ You can install the dependencies by running:
 
     sudo apt-get install build-essential erlang-base \
         erlang-dev erlang-manpages erlang-eunit erlang-nox \
-        libicu-dev libmozjs185-dev libcurl4-openssl-dev \
-        pkg-config
+        erlang-reltool libicu-dev libmozjs185-dev libcurl4-openssl-dev \
+        pkg-config rebar
```

Comment:
Master doesn't requires rebar as [we use our own ](https://github.com/apache/couchdb/blob/master/configure#L216-L225). To use system one, you need to explicitly define REBAR environment variable with the path to yours for running ./configure and make commands.